src/main/java/com/epam/processor/DataProcessor.java

```diff
@@ -40,7 +45,15 @@ public RoadAccident getAccidentByIndex7(String index){
      * @return
      */
     public Collection<RoadAccident> getAccidentsByLocation7(float minLongitude, float maxLongitude, float minLatitude, float maxLatitude){
-        return null;
+        List<RoadAccident> result = new ArrayList<>();
+        for (RoadAccident rec : roadAccidentList ) {
+            float longitude = rec.getLongitude();
+            float latitude = rec.getLatitude();
+            if (longitude>=minLongitude && longitude<=maxLongitude && latitude>=minLatitude && latitude<=maxLatitude )
```

Comment:
I would recommend to extract coordinates checking logic into separated method. Something like isInScope... 

Again - we have to make our code easy to read, right?